server/handlers.go

```diff
@@ -755,11 +748,9 @@ func (s *Server) purgeHandler() {
 	ticker := time.NewTicker(s.purgeInterval)
 	go func() {
 		for {
-			select {
-			case <-ticker.C:
-				err := s.storage.Purge(s.purgeDays)
-				s.logger.Printf("error cleaning up expired files: %v", err)
-			}
+			<-ticker.C
+			err := s.storage.Purge(s.purgeDays)
+			s.logger.Printf("error cleaning up expired files: %v", err)
```

Comment:
let's add an `err != nil` in order to log, what do you think?

Reply:
Done